composer.json

```diff
@@ -71,7 +71,8 @@
     "psr/cache": "~1.0",
     "psr/container": "^1.0.0",
     "psr/simple-cache" : "^1.0.1",
-    "ramsey/uuid": "^3.8"
+    "ramsey/uuid": "^3.8",
+    "composer-runtime-api": "^2.0"
```

Comment:
Composer 2 must available consistently prior the merge and release of these changes.